src/align_assign.cpp

```diff
@@ -31,8 +31,10 @@ chunk_t *align_assign(chunk_t *first, size_t span, size_t thresh, size_t *p_nl_c
    }
    size_t my_level = first->level;
 
+   char   *copy = (char *)malloc(1000);
+
    LOG_FMT(LALASS, "%s(%d): [my_level is %zu]: start checking with '%s', on orig_line %zu, span is %zu, thresh is %zu\n",
-           __func__, __LINE__, my_level, first->text(), first->orig_line, span, thresh);
+           __func__, __LINE__, my_level, first->text_first_999(copy), first->orig_line, span, thresh);
```

Comment:
What about a more generic name, maybe something like `short_text` or `elided_text`? (Especially since a) the returned text *isn't* necessarily truncated, nor b) likely to actually be 999 characters...)